### ISSUE TITLE
Mint failure for test 'PresignedGetObject_Test1' against Minio in FS mode

### ISSUE THREAD

**Issue by ebozduman** [minio/minio-dotnet#244]
Minio version: 
Version: 2018-09-10T23:25:15Z
Release-Tag: DEVELOPMENT.2018-09-10T23-25-15Z
Commit-ID: 75715820007a8832f4bc86c338d2ce1cb11eff2f

Mint ("edge" as of 09/10/2018) failed for minio-dotnet test with the following error message:
```
$ sudo docker run -e ACCESS_KEY=minio -e SECRET_KEY=minio123 -e SERVER_ENDPOINT=<ip_address>:9000 -e ENABLE_HTTPS="0" -e SERVER_REGION="us-east-1" -e MINT_MODE="full" --net=host minio/mint:edge
Running with
SERVER_ENDPOINT:      192.168.1.124:9000
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp aaddee5d2484:/mint/log /tmp/mint-logs'
(1/13) Running aws-sdk-go tests ... done in 0 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-php tests ... done in 43 seconds
(4/13) Running aws-sdk-ruby tests ... done in 1 seconds
(5/13) Running awscli tests ... done in 20 seconds
(6/13) Running mc tests ... done in 9 seconds
(7/13) Running minio-dotnet tests ... ./run.sh: line 26:  1180 Aborted                 (core dumped) /mint/run/core/minio-dotnet/out/Minio.Functional.Tests >> "$output_log_file" 2> "$error_log_file"
FAILED in 2 minutes and 15 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (The operation has timed out.) ---> System.Net.WebException: The operation has timed out.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Minio.Functional.Tests.FunctionalTest.PresignedGetObject_Test1(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 1768
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.Main(String[] args) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 234

Executed 6 out of 13 tests successfully.
```
/mint/run/core/minio-dotnet/FunctionalTest.cs:line 1768 points to task `PresignedGetObject_Test1`.

Reproducibility:
Running the same mint:edge tests against the same minio version, the failure is reproducible.

**Comment by kannappanr** [comment]
Succeeds with 08-25 build and fails with 09-01 build

**Comment by poornas** [comment]
This test failure is due to regression from commit 72fa2b4537da746236541eb0592e7de3800a60b4(https://github.com/minio/minio/pull/6206). The PresignedGetObject test fails because the http listener times out while getting resource in line 207-213 of cmd/http/listener.go.

**Comment by poornas** [comment]
This is fixed by https://github.com/minio/minio/pull/6465

**Comment by kannappanr** [comment]
Closing this issue as fixed based on minio/minio#6465
